middleware/metrics.ts

```diff
@@ -32,7 +34,17 @@ export function createMetricsMiddleware() {
       const { routePath } = req;
       const { statusCode } = res;
 
-      const path = routePath || parse(url).pathname || "<NONE>";
+      let path = routePath;
+      if (!path) {
+        if (statusCode === 404) {
+          path = "<NOT_FOUND>";
+        } else if (opts.trackAllFoundPaths) {
+          path = parse(url).pathname ?? "<UNKNOWN>";
+        } else {
+          path = "<UNTRACKED>";
```

Comment:
Should this be <NONE> to keep consistency with the previous implementation?

Reply:
I'm guessing you mean should it be `<NONE>`? Don't think it really needs to be consistent with the existing labels